Formula/lensfun.rb

```diff
@@ -22,6 +22,11 @@ class Lensfun < Formula
   depends_on "libpng"
   depends_on "python@3.9"
 
+  patch do
+    url "https://github.com/lensfun/lensfun/commit/de954c9.patch?full_index=1"
```

Comment:
```suggestion
    url "https://github.com/lensfun/lensfun/commit/de954c952929316ea2ad0f6f1e336d9d8164ace0.patch?full_index=1"
```
Let's use the full commit hash here (it's why we use `full_index` too). Can you also include a comment indicating when this patch can be removed?

Please squash this change with your first commit. Thanks!

Reply:
Thank you very much!  I've just done it.